pygls/uris.py

```diff
@@ -29,6 +29,9 @@
 
 
 def _normalize_win_path(path):
+    if path is None:
+        return None
```

Comment:
Should this return `None, None`?

Reply:
Good catch. I will remove it completely and catch error where this function is called.